src-frontend/app/controllers/procedure.js

```diff
@@ -63,6 +64,29 @@ export default Ember.ObjectController.extend({
             });
 
             this.endPropertyChanges();
+        },
+        generateProtocol: function() {
+            var filename = this.get('model.title') + '.xml';
+            $.ajax({
+                beforeSend: function(xhr) {
+                    xhr.setRequestHeader('Authorization', 'Token ' + Cookies.get('authorizationToken'));
```

Comment:
Yeah I think we were going to hold off on that until we get a chance to evaluate [ember-simple-auth](https://github.com/simplabs/ember-simple-auth)